.github/workflows/ci.yml

```
@@ -33,9 +33,32 @@ jobs:
+    - name: Test build
+      run: |
+        #docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_PAT }} ghcr.io 
+        docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io  
+        docker build . --file ./ubuntu/latest.Dockerfile --tag "ubuntu-latest-latest"
+        docker images
+        # docker tag ghcr.io/lecaqquochung/ubuntu-latest:latest ghcr.io/lecaqquochung/ubuntu-latest:latest
+        # docker push ghcr.io/lecaoquochung/docker-images:ubuntu-latest-latest
     - name: ubuntu-lastest
-      run: docker build . --file ./ubuntu/latest.Dockerfile --tag "ubuntu-latest:$(date +%s)"
-  
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1101:error:4:16: Delete trailing spaces after \ to break line (or use quotes for literal space)

(shellcheck)